src/components/FileTree/index.tsx

```diff
@@ -179,9 +210,28 @@ export class FileTreeBase extends React.Component<Props> {
           isNodeExpanded={this.isNodeExpanded}
           onToggleExpand={this.onToggleExpand}
         />
+        <button id="openAll" onClick={this.onExpandTree} type="button">
```

Comment:
Always use bootstrap components to keep the look and feel consistent.

I have a patch to suggest (below):
* Move the buttons up top where they are more effective. Without this, the *close all* button is inaccessible after opening everything.
* Render small, light buttons side by side to conserve space
* Truncate the button text on small screens. This just looks nicer but, also, *Open all ...* still makes enough sense.

```diff
diff --git a/src/components/FileTree/index.tsx b/src/components/FileTree/index.tsx
index 947dece..de12414 100644
--- a/src/components/FileTree/index.tsx
+++ b/src/components/FileTree/index.tsx
@@ -1,8 +1,9 @@
 import * as React from 'react';
-import { ListGroup } from 'react-bootstrap';
+import { Button, ListGroup } from 'react-bootstrap';
 import { connect } from 'react-redux';
 import Treefold, { TreefoldRenderProps } from 'react-treefold';
 
+import styles from './styles.module.scss';
 import FileTreeNode, {
   PublicProps as FileTreeNodeProps,
 } from '../FileTreeNode';
@@ -204,18 +205,34 @@ export class FileTreeBase extends React.Component<Props> {
 
     return (
       <ListGroup>
+        <div className={styles.controlButtons}>
+          <Button
+            className={styles.button}
+            id="openAll"
+            onClick={this.onExpandTree}
+            size="sm"
+            type="button"
+            variant="light"
+          >
+            {gettext('Open all folders')}
+          </Button>
+          <Button
+            className={styles.button}
+            id="closeAll"
+            onClick={this.onCollapseTree}
+            size="sm"
+            type="button"
+            variant="light"
+          >
+            {gettext('Close all folders')}
+          </Button>
+        </div>
         <Treefold
           nodes={[tree]}
           render={this.renderNode}
           isNodeExpanded={this.isNodeExpanded}
           onToggleExpand={this.onToggleExpand}
         />
-        <button id="openAll" onClick={this.onExpandTree} type="button">
-          {gettext('Open all folders')}
-        </button>
-        <button id="closeAll" onClick={this.onCollapseTree} type="button">
-          {gettext('Close all folders')}
-        </button>
       </ListGroup>
     );
   }
diff --git a/src/components/FileTree/styles.module.scss b/src/components/FileTree/styles.module.scss
new file mode 100644
index 0000000..01fb104
--- /dev/null
+++ b/src/components/FileTree/styles.module.scss
@@ -0,0 +1,14 @@
+.controlButtons {
+  display: grid;
+  grid-column-gap: 10px;
+  grid-template-columns: 1fr 1fr;
+  margin-bottom: 10px;
+  width: 100%;
+}
+
+.button {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+}
```

Reply:
Thanks, that looks awesome!